examples/l0-token/build.sbt

```diff
@@ -0,0 +1,32 @@
+import Dependencies._
+
+resolvers += Resolver.mavenLocal
+
+ThisBuild / scalaVersion := "2.13.6"
+ThisBuild / version := "0.1.0-SNAPSHOT"
+ThisBuild / organization := "com.example"
```

Comment:
I just realized that the scala packages are `org.example`. Could you change it here to `org` instead of `com` and do the same thing also in the empty state channel `build.sbt`?